recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
   sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
-    - python.app  # [osx]
```

Comment:
I don't know whether these should be dependencies for ``sip``. ``python.app`` should certainly be a dependency for any Qt package such as Spyder or glue, but I'm just not sure if the dependency has to be defined at this level (in sip)